guides/source/getting_started.md

```diff
@@ -61,6 +61,20 @@ The Rails philosophy includes two major guiding principles:
   things in a web application, and defaults to this set of conventions, rather than
   require that you specify minutiae through endless configuration files.
 
+Developer environment and editor integrations
```

Comment:
For [headings,](https://guides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings), we capitalize all words except prepositions, conjunctions, internal articles, and forms of the verb "to be":
```suggestion
Developer Environment and Editor Integrations
```